R/tokens.R

```diff
@@ -483,7 +483,7 @@ compile_removals_regex <- function(remove_separators = FALSE,
     }
     if (remove_numbers) {
         # includes currency amounts and those containing , or . digit separators
-        regex_to_remove <- c(regex_to_remove, "^\\p{Sc}{0,1}([0-9]{1,3}([,.][0-9]{3})*([,.][0-9]+)?|[.,][0-9]+)\\p{Sc}{0,1}$")
+        regex_to_remove <- c(regex_to_remove, "^\\p{Sc}{0,1}([0-9]{1,3}([,.][0-9]{3})*([,.][0-9]+)?|[.,][0-9]+)\\p{Sc}{0,1}$|^\\d+$")
```

Comment:
How about sicking to Unicode character class `\\p{N}`? Also how about simplifying it to `"^\\p{Sc}?[,.\\p{N}]+$"`?

Reply:
Simplifying is good, I agree, so is using the Unicode N character class. But we need also to check that:
- the `,` or `.` are thousands or decimal places, properly formed (groups of three unless a decimal)
- currency symbol can come at the end as well